src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/Target_X64/X64ReadyToRunHelperNode.cs

```diff
@@ -150,7 +150,6 @@ protected override void EmitCode(NodeFactory factory, ref X64Emitter encoder, bo
                         }
                         else
                         {
-                            ISymbolNode targetMethodNode = target.GetTargetNode(factory);
```

Comment:
Might be good to do this kind of changes in the upstream repo because the only reward you get for this good deed (for the time being) is a potential for new merge conflicts.

Reply:
Right, now in #1840